Converters/Revit/Speckle.Converters.RevitShared/ToHost/TopLevel/ToDirectShape/CurveToDirectShapeConverterToHostBase.cs

```diff
@@ -33,7 +33,7 @@ public List<DB.GeometryObject> Convert(TCurve target)
       DB.BuiltInCategory.OST_GenericModel
     );
 
-    var revitDs = DB.DirectShape.CreateElement(_contextStack.Current.Document, genericModelCategory.Id);
+    using var revitDs = DB.DirectShape.CreateElement(_contextStack.Current.Document, genericModelCategory.Id);
```

Comment:
@connorivy in a few places where we are creating revit elements, but they are falling out of scope (e.g. we are not returning them, or passing them into another object),
I'm adding a `using` keyword to dispose the C# object.

In my testing, this appears to work fine, the object still converts as expected. Adding the `using` keywords might help us be deterministic with disposal, and perhaps lower our memory footprint.

The alternative is we start silencing CA2000 just for Revit, but I don't see any downsides to what I'm doing here.

Reply:
I'm not aware of any reason this wouldn't be okay